foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -116,8 +119,10 @@ public static ConcurrentCompositeConfiguration createLocalConfig() {
     }
 
     LOGGER.info("create local config:");
-    for (ConfigModel configModel : loader.getConfigModels()) {
-      LOGGER.info(" {}.", configModel.getUrl());
+    boolean isPrintUrl = DynamicPropertyFactory.getInstance()
+        .getBooleanProperty(IS_PRINT_URL, true).get();
+    if (isPrintUrl) {
+      LOGGER.info(" {}.", loader.getConfigModels());
```

Comment:
load.getConfigModels is a List.  It's a bad practise to invoke toString of a List. Maybe
```
StringUtils.join(loader.getConfigModels(), ",") 
```
is better.

Reply:
done.